schema-loader/src/integration-test/java/com/scalar/db/schemaloader/cassandra/CassandraSchemaLoaderIntegrationTest.java

```diff
@@ -0,0 +1,43 @@
+package com.scalar.db.schemaloader.cassandra;
+
+import com.google.common.collect.ImmutableList;
+import com.scalar.db.config.DatabaseConfig;
+import com.scalar.db.schemaloader.SchemaLoaderIntegrationTestBase;
+import com.scalar.db.storage.cassandra.CassandraEnv;
+import java.io.FileOutputStream;
+import java.util.List;
+import java.util.Properties;
+
+public class CassandraSchemaLoaderIntegrationTest extends SchemaLoaderIntegrationTestBase {
+  private static final DatabaseConfig config = CassandraEnv.getDatabaseConfig();
+
+  @Override
+  protected void initialize() throws Exception {
+    Properties properties = config.getProperties();
+    try (final FileOutputStream fileOutputStream = new FileOutputStream(CONFIG_FILE)) {
+      properties.store(fileOutputStream, null);
+    }
```

Comment:
It looks like we can move this logic to the Base class?

Reply:
I moved it to the base class [commit](https://github.com/scalar-labs/scalardb/pull/349/commits/aa0a087c02f328990bf34f07275bbc3730461a9f)